src/api_client.rs

```diff
@@ -1,560 +1,596 @@
-use ethereum_consensus::altair::mainnet::{
-    SignedContributionAndProof, SyncCommitteeContribution, SyncCommitteeMessage,
-};
-use ethereum_consensus::bellatrix::mainnet::{BlindedBeaconBlock, SignedBlindedBeaconBlock};
-use ethereum_consensus::networking::{Enr, MetaData, Multiaddr, PeerId};
-use ethereum_consensus::phase0::mainnet::{
-    Attestation, AttestationData, AttesterSlashing, BeaconBlock, BeaconState, Checkpoint, Fork,
-    ProposerSlashing, SignedAggregateAndProof, SignedBeaconBlock, SignedBeaconBlockHeader,
-    SignedVoluntaryExit, Validator,
-};
-use ethereum_consensus::primitives::{
-    BlsPublicKey, Bytes32, ChainId, CommitteeIndex, Coordinate, Epoch, ExecutionAddress, Gwei,
-    RandaoReveal, Root, Slot, ValidatorIndex, Version,
-};
+// use ethereum_consensus::altair::mainnet::{
```

Comment:
not sure why you need to drop all the imports...

Reply:
rustup update seems to have fixed this